desktop/panel/database.test.js

```diff
@@ -126,16 +141,24 @@ for (const subprocess of RUNNERS) {
             { age: '20', name: 'Bake', location: { city: 'Toronto' } },
           ]);
 
+          const databaseConnectorConfig = {
+            type: t.type,
+            database: vendorOverride[t.type]?.database || 'test',
+            address: vendorOverride[t.type]?.address || 'localhost',
+            username: vendorOverride[t.type]?.username || 'test',
+            password_encrypt: new Encrypt(
+              vendorOverride[t.type]?.password || 'test'
+            ),
+          };
+
+          if (t.type === 'odbc') {
+            databaseConnectorConfig.extra = {
+              driver: 'ODBC Driver 18 for SQL Server',
```

Comment:
Not bad but just to keep overrides together you can just add a new optional `extra` field where you define the odbc overrides in line 100.

Just make line 100:

```
odbc: {
  address: 'localhost'
  ...
  database: 'master',
 extra: {
  driver: 'ODBC .. Server'
 },
}
```

And then pass in `extra` to the DatabaseConnectorInfo (I think it takes `extra` as a config parameter.